refinery/README.md

```diff
@@ -0,0 +1,165 @@
+  Sampler: DeterministicSampler
+  SampleRate: 1
+```
+
+To configure custom sampling rules, you will need to pass in your own `refinery.yaml` file.
+[The Refinery documentation](https://docs.honeycomb.io/manage-data-volume/refinery/sampling-methods/) goes into more detail about how each sampling method works.
+These example configurations are provided to demonstrate how to define rules in YAML.
+
+### Rules-Based Sampler
+
+```yaml
+# refinery.yaml
+rules:
+#  DryRun: false
+  my_dataset:
+    Sampler: "RulesBasedSampler"
+    rule:
+      - name: "keep 5xx errors"
+        SampleRate: 1
+        condition:
+          field: "status_code"
+          operator: ">="
+          value: 500
+      - name: "downsample 200 responses"
+        SampleRate: 1000
+        condition:
+          field: "status_code"
+          operator: "="
+          value: 200
+      - name: "send 1 in 10 traces"
+        SampleRate: 10 # base case
+```
+
+### Exponential Moving Average (EMA) Dynamic Sampler
+
+This is the recommended sampling method for most teams.
+
+```yaml
+# refinery.yaml
+rules:
+#  DryRun: false
+  my_dataset:
+    Sampler: "EMADynamicSampler"
+    GoalSampleRate: 2
+    FieldList:
+      - "request.method"
+      - "response.status_code"
+   AdjustmentInterval: 15
+   Weight: 0.5
+```
+
+## Configuration
+
+The repository's [values.yaml](./values.yaml) file contains information about all configuration options for this chart.
+
+### Refinery Metrics
+
+If you decide to send [Refinery's runtime metrics](https://docs.honeycomb.io/manage-data-volume/refinery/scale-and-troubleshoot/#understanding-refinerys-metrics) to Honeycomb, you will need to give Refinery an API for the team you want those metrics sent to.
+
+You can obtain your API Key by going to your Account profile page inside of your Honeycomb instance.
+
+```yaml
+# refinery.yaml
+config:
+  Metrics: honeycomb
+  HoneycombMetrics:
+    # MetricsHoneycombAPI: https://api.honeycomb.io # default
+    MetricsAPIKey: YOUR_API_KEY
+    # MetricsDataset: "Refinery Metrics" # default
+    # MetricsReportingInterval: 3 # default
+```
+
+## Parameters
+
+The following table lists the configurable parameters of the Refinery chart and their default values, as defined in [`values.yaml`](./values.yaml).
+
+| Parameter | Description | Default |
+| --- | --- | --- |
+| `replicaCount` | Number of Refinery replicas | `2` |
+| `image.repository` | Refinery image name | `honeycombio/refinery` |
+| `image.pullPolicy` | Refinery image pull policy | `IfNotPresent` |
+| `image.tag` | Refinery image tag (leave blank to use app version) | `nil` |
+| `imagePullSecrets` | Specify docker-registry secret names as an array | `[]` |
+| `nameOverride` | String to partially override refinery.fullname template with a string (will append the release name) | `nil` |
+| `fullnameOverride` | String to fully override refinery.fullname template with a string | `nil` |
+| `config.ListenAddr` | blah | `0.0.0.0:8080` |
```

Comment:
I just realized a lot of these are in the refinery docs, so I'll go ahead and update those.